modules/dreamview/backend/simulation_world/simulation_world_service.cc

```diff
@@ -1013,6 +1013,14 @@ void SimulationWorldService::UpdateSimulationWorld(
 template <>
 void SimulationWorldService::UpdateSimulationWorld(
     const NavigationInfo &navigation_info) {
+  static double previous_timestamp = 0.0;
+  double topic_timestamp = navigation_info.header().timestamp_sec();
+  if (previous_timestamp == topic_timestamp) {
+    // it seems to be the same topic, ignore this update
+    return;
+  }
```

Comment:
Not safe to compare float numbers to be exactly equal.

Reply:
Fixed, please review again. THX.^^